tests/benchmarks/test_evaluate.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from .fixtures import _complicated_page
+
+
+@pytest.mark.benchmark
+def test_evaluate_page():
+    _complicated_page()
```

Comment:
i also think we should parametrize this one pre-emptively. even though we only have one case now, if we parametrize it, then it's very easy to add additional pages to evaluate and the naming structure of the test case wouldn't change, the new ones would just be seen as new benchmarks.